android/KMEA/app/src/main/AndroidManifest.xml

```diff
@@ -68,6 +68,16 @@
             android:label="@string/app_name"
             android:theme="@style/Theme.AppCompat.Light.Dialog" >
         </activity>
+
+      <provider
+        android:name="androidx.core.content.FileProvider"
+        android:authorities="${applicationId}.fileProvider"
+        android:exported="false"
+        android:grantUriPermissions="true">
+        <meta-data
+          android:name="android.support.FILE_PROVIDER_PATHS"
+          android:resource="@xml/file_paths" />
+      </provider>
```

Comment:
In #2053 we intentionally reverted FileProvider from KMEA back to the main KMAPro. (It had namespace conflicts with the First Voices apps).

Is this needed for one of the new tests?

Reply:
I removed the file provider and add some Hotfixes to the Helplink impl for testmode.

I supposed a permission problem, when my helplink test run into a timeout. After fixing the deadlock the tests runs without the fileprovider.